tests/ui/closures/issue-72408-nested-closures-exponential.rs

```diff
@@ -1,5 +1,5 @@
 //@ build-pass
-//@ ignore-compare-mode-next-solver (hangs)
+//@ ignore-test (hangs)
```

Comment:
😊 should probably use a FIXME

Reply:
well this should be fixed before this lands, dont think we can land in this state bc it will definitely break a real crate